src/main/kotlin/org/rust/ide/annotator/RustHighlightingAnnotator.kt

```diff
@@ -96,5 +96,8 @@ class RustHighlightingAnnotator : Annotator {
         override fun visitTraitMethodMember(o: RustTraitMethodMemberElement) =
             highlight(o.identifier, if (o.isStatic) RustColor.STATIC_METHOD else RustColor.INSTANCE_METHOD)
 
+        override fun visitParameters(o: RustParametersElement) = o.parameterList.forEach { highlight(it.pat, RustColor.PARAMETER) }
```

Comment:
I think this is not exactly what we want here, because a single pattern can introduce multiple bindings, each of which should be colored separately: `fn foo((x, y): (i32, i32))`. Perhaps we can do something like

```
when {
    o.parentOfType<RustParameterElement>() != null ->
        highlight(o, RustColor.PARAMETER)

    o.isMut ->
        highlight(o.identifier, RustColor.MUT_BINDING)
}
```

?

That way, the usages of function parameters will be colored as well!